content/blog/2022/2022-01-25-annotation-processing.md

```diff
@@ -0,0 +1,953 @@
+---
+title: "An Introduction to Annotations and Annotation Processing"
+categories: ["Java"]
+date: 2022-01-27 00:00:00 +1100 
+modified: 2022-01-27 00:00:00 +1100
+authors: [syedaf]
```

Comment:
Please add yourself to the file `data/authors.yml` under the name `syedaf` so you appear as the author. If you have a headshot, please also add a photo (the photo will currently not be shown, but I'm going to add an author page that shows it, soon)